test/karma/bool.cpp

```diff
@@ -117,7 +117,7 @@ int main()
 
         bool b = false;
         BOOST_TEST(test("false", bool_, phoenix::ref(b)));
-        BOOST_TEST(test("true", bool_, ++phoenix::ref(b)));
+        BOOST_TEST(test("true", bool_, !phoenix::ref(b)));
```

Comment:
At least this works for me on GCC and Clang https://travis-ci.org/Kojoley/spirit/jobs/155156461
```
common.mkdir ../../../bin.v2/libs/spirit/test/karma_bool-p3.test
common.mkdir ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8
common.mkdir ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug
common.mkdir ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on
common.mkdir ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on/karma
clang-linux.compile.c++.without-pth ../../../bin.v2/libs/spirit/test/karma_binary3-p3.test/clang-linux-3.8/debug/coverage-on/karma/binary3.o
clang-linux.link ../../../bin.v2/libs/spirit/test/karma_binary3-p3.test/clang-linux-3.8/debug/coverage-on/karma_binary3-p3
testing.capture-output ../../../bin.v2/libs/spirit/test/karma_binary3-p3.test/clang-linux-3.8/debug/coverage-on/karma_binary3-p3.run
**passed** ../../../bin.v2/libs/spirit/test/karma_binary3-p3.test/clang-linux-3.8/debug/coverage-on/karma_binary3-p3.test
common.mkdir ../../../bin.v2/libs/spirit/test/karma_buffer-p3.test
common.mkdir ../../../bin.v2/libs/spirit/test/karma_buffer-p3.test/clang-linux-3.8
common.mkdir ../../../bin.v2/libs/spirit/test/karma_buffer-p3.test/clang-linux-3.8/debug
common.mkdir ../../../bin.v2/libs/spirit/test/karma_buffer-p3.test/clang-linux-3.8/debug/coverage-on
common.mkdir ../../../bin.v2/libs/spirit/test/karma_buffer-p3.test/clang-linux-3.8/debug/coverage-on/karma
clang-linux.compile.c++.without-pth ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on/karma/bool.o
clang-linux.link ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on/karma_bool-p3
testing.capture-output ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on/karma_bool-p3.run
**passed** ../../../bin.v2/libs/spirit/test/karma_bool-p3.test/clang-linux-3.8/debug/coverage-on/karma_bool-p3.test
```